source/features/action-used-by-link.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+import searchIcon from 'octicon/search.svg';
+
+function init(): void {
+	const actionRepo = (select('.py-3.border-bottom.border-gray-light .octicon.octicon-issue-opened.text-gray-dark.mr-2')!.parentElement as HTMLAnchorElement)
```

Comment:
JMHO
```suggestion
	const actionRepo = (select('.octicon-issue-opened.text-gray-dark')!.parentElement as HTMLAnchorElement)
		.pathname.slice(1)
		.replace('/issues', '');
```
Its rare that github styles their own icons (from my experience)

Reply:
`.slice(1)` drops the initial slash so `/issues` will never match. You only need one of them anyway, either slice or replace, they do the same thing.

---

But really, why use the issue-opened icon and then removing `/issues`? There's already a plain link, you used it before.

---

Also try avoiding to use utility classes (`.mt-1`, `.text-gray-dark`) when selecting items; they are more prone to change or likely to appear in new places.

```suggestion
	const actionRepo = (select('aside .octicon-repo')!.parentElement as HTMLAnchorElement)
		.pathname.slice(1);
```

If this works, even better:


```suggestion
	const actionRepoLink: HTMLAnchorElement = select('aside .octicon-repo')!.parentElement!;
	const actionRepoURI = actionRepoLink.pathname.slice(1);
```

Reply:
`.slice(1)` drops the initial slash so `/issues` will never match. You only need one of them anyway, either slice or replace, they do the same thing.

---

But really, why use the issue-opened icon and then removing `/issues`? There's already a plain link, you used it before.

---

Also try avoiding to use utility classes (`.mt-1`, `.text-gray-dark`) when selecting items; they are more prone to change or likely to appear in new places.

```suggestion
	const actionRepo = (select('aside .octicon-repo')!.parentElement as HTMLAnchorElement)
		.pathname.slice(1);
```

If this works, even better:


```suggestion
	const actionRepoLink: HTMLAnchorElement = select('aside .octicon-repo')!.parentElement!;
	const actionRepoURI = actionRepoLink.pathname.slice(1);
```